src/Makefile.am

```diff
@@ -246,6 +246,22 @@ libeos_shell_js_la_CPPFLAGS =		\
 
 ########################################
 
+# Set up headers for libeos-shell-fx
+eos_shell_fxincludedir = $(includedir)
+eos_shell_fxinclude_HEADERS = wobbly-effect.h
+
+eos_shell_fx_cppflags = $(AM_CPPFLAGS) $(EOS_SHELL_FX_CFLAGS) -std=c++11 -Werror -Wall -Wextra -Wno-error=unused-parameter -Wno-error=deprecated-declarations
```

Comment:
I don't want one part of the project to use different flags than the rest... Please remove them, but keep the C++11 flag if needed. (Or otherwise, find a way to apply them to all the project instead, but I don't think it's worth investing in that direction given that upstream now does just that -- https://git.gnome.org/browse/gnome-shell/tree/configure.ac#n225)